src/BaGet.Core/Authentication/AuthenticationHeaderValueExtensions.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Net;
+using System.Net.Http.Headers;
+using System.Text;
+
+namespace BaGet.Core.Authentication
+{
+    public class AuthenticationHeaderValueExtensions
+    {
+        public static string GetBase64BasicAuthHeaderString(NetworkCredential credential)
```

Comment:
From the class name it seems that these should be extension methods on the `NetworkCredential` type. I think this pattern would be pretty nice:

```
NetworkCredential credentials = ...;

credentials.ToBase64BasicAuthHeaderString();
credentials.ToBasicAuthHeaderString();`